### DISCUSSION TITLE
[Feature Request]: Power Platform CLI Extensions

### DISCUSSION THREAD

**Discussion by rajyraman** [microsoft/powerplatform-vscode#307] [Ideas]
Since there is no discussions tab, I am logging this as an issue, but this is a feature request.

GitHub CLI has an [extensibility model](https://docs.github.com/en/github-cli/github-cli/creating-github-cli-extensions). It would be great if Power Platform CLI has an extensibility model as well so that community can create additional commands and people can install this as third-party commands.

cc @devkeydet

**Comment by davidjenni** [comment]
@rajyraman well then, let's add a discussions tab to this repo :-)

The pac CLI was designed with an architecture to support extensibility (using MEF), but the current API we're using is not stable yet for external usage. It is on our roadmap to support the scenario to discover and load nouns/verbs that are not shipped/supported by MSFT but are valuable contributions by the community

CC: @kkanakas
